physics/MP/GFDL/module_gfdlmp_param.F90

```diff
@@ -0,0 +1,413 @@
+  real(kind_phys) :: rewmin         = 5.0     !< (v1/v3) minimum effective radii (liquid)
+  real(kind_phys) :: reimin         = 10.0    !< (v1/v3) minimum effective radii (ice)
+  real(kind_phys) :: reimax         = 150.0   !< (v1/v3) maximum effective radii (ice)
+  real(kind_phys) :: rermax         = 10000.0 !< (v1/v3) maximum effective radii (rain)
+  real(kind_phys) :: resmin         = 150.0   !< (v1/v3) minimum effective radii (snow)
+  real(kind_phys) :: resmax         = 10000.0 !< (v1/v3) maximum effective radii (snow)
+  real(kind_phys) :: regmax         = 10000.0 !< (v1/v3) maximum effective radii (graupel)
+  !
+  logical :: const_vi               = .false. !< (v1/v3) if .t. the constants are specified by v * _fac
+  logical :: const_vs               = .false. !< (v1/v3) if .t. the constants are specified by v * _fac
+  logical :: const_vg               = .false. !< (v1/v3) if .t. the constants are specified by v * _fac
+  logical :: const_vr               = .false. !< (v1/v3) if .t. the constants are specified by v * _fac
+  logical :: z_slope_liq            = .true.  !< (v1/v3) use linear mono slope for autocconversions
+  logical :: do_hail                = .false. !< (v1/v3) use hail parameters instead of graupel
+  logical :: do_qa                  = .true.  !< (v1/v3) do inline cloud fraction
+  logical :: rad_snow               = .true.  !< (v1/v3) consider snow in cloud fraciton calculation
+  logical :: rad_graupel            = .true.  !< (v1/v3) consider graupel in cloud fraction calculation
+  logical :: rad_rain               = .true.  !< (v1/v3) consider rain in cloud fraction calculation
+  logical :: do_sedi_heat           = .true.  !< (v1/v3) transport of heat in sedimentation
+  logical :: prog_ccn               = .false. !< (v1/v3) do prognostic ccn (yi ming's method)
+  logical :: tintqs                 = .false. !< (v1/v3)
+  !
+  integer :: icloud_f               = 0       !< (v1/v3) GFDL cloud scheme
+                                              !< 0: subgrid variability based scheme
+                                              !< 1: same as 0, but for old fvgfs implementation
+                                              !< 2: binary cloud scheme
+                                              !< 3: extension of 0
+  integer :: irain_f                = 0       !< (v1/v3) cloud water to rain auto conversion scheme
+                                              !< 0: subgrid variability based scheme
+                                              !< 1: no subgrid varaibility
+
+  ! #####################################################################################
+  ! GFDL MP common (v1/v3) parameters, with different default values
+  ! #####################################################################################
+#ifdef GFDLMP_V3
+  real(kind_phys) :: tice           = 273.15  !< freezing temperature (K): ref: GFDL, GFS (DJS: V3=273.15)
```

Comment:
Why does this need to be redefined from the one that is already provided by the host (temperature_at_zero_celsius)? Does it really make that much of a difference to 1) use a different value for each version and 2) use something different than the host and rest of physics?

Reply:
@grantfirl These are the defaults and (potentially) overwritten later on. I don't know if the 0.01K difference is by design or an oversight (I didn't want to add any changes when creating this module).
@RuiyuSun Does v3 need tice=273.16? or could we use 273.15?

Reply:
@dustinswales In v3 it is defined as 273.15.

Reply:
According to the comment in the v3 code, 273.15 is used to be consistent with the value defined in other schemes in GFS